app/assets/scss/app-old-ie.scss

```diff
@@ -0,0 +1,5 @@
+// If you want to display the currently active breakpoint in the top
```

Comment:
Is there a reason these aren't using the [existing app stylesheets](https://github.com/alphagov/govuk-frontend/tree/ad2022ef044d2e67768199479e215f641e36cc5f/app/assets/scss), which are now not being used?

Reply:
the task to compile css for dist and review app is the same. tried using the same entry point for both and couldn't find a way to not include breakpoints in dist compiled and include them in review compiled

Reply:
govuk-frontend*.scss fiels are used for compilation to dist

Reply:
The `govuk-frontend*.scss` files **in the app folder** are used for compilation to dist?

Reply:
yes. happy to find a better place for them.
they used to be in globals and we were filtering them from copy task.